.github/workflows/pack-nsis.yml

```diff
@@ -155,6 +140,10 @@ jobs:
     permissions:
       contents: write
     steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Check the tag consistency with the source version
```

Comment:
Check this in the `build` job if the workflow is launched on a Git tag (see above)?

Reply:
Ok & Done as above